apps/api/src/app/environments-v2/usecases/diff-environment/diff-environment.usecase.ts

```diff
@@ -40,7 +42,7 @@ export class DiffEnvironmentUseCase {
        * For now, we only support workflow diff
        * In the future, we can add more strategies here
        */
-      const strategies = [this.workflowSyncStrategy];
+      const strategies = [this.workflowSyncStrategy, this.layoutSyncStrategy];
```

Comment:
I wonder in terms of sequence and dependency, if it's better to promote layouts before workflows 🤔

Reply:
Oh I see, makes sense.